Projects/Server/Network/Packets/IncomingAccountPackets.cs

```diff
@@ -185,6 +188,143 @@ public static void CreateCharacter(NetState state, CircularBufferReader reader,
         }
     }
 
+    public static void KRCreateCharacter(NetState state, CircularBufferReader reader, int packetLength)
+    {
+        int flags = 0;
+
+        int length = packetLength;
+
+        int unk1 = reader.ReadInt32(); // Pattern
```

Comment:
Change comment to `// 0xEDEDEDED`
Since I think that is the constant?

Reply:
Solved